src/ert/scheduler/job.py

```diff
@@ -81,6 +80,9 @@ async def __call__(
             while not self.returncode.done():
                 await asyncio.sleep(0.01)
             returncode = await self.returncode
+            # we need to make sure that the task has finished too
+            await self.driver.wait(self.real.iens)
```

Comment:
I think the presence of a comment here means that the function in the driver should have a clearer name.

Reply:
Right, that was not clear that the name was an issue :)